.aws/task-definition.json

```diff
@@ -21,33 +21,33 @@
       "logConfiguration": {
         "logDriver": "awslogs",
         "options": {
-          "awslogs-group": "awslogs-tis-template",
+          "awslogs-group": "awslogs-tis-revalidation-concerns",
           "awslogs-region": "eu-west-2",
-          "awslogs-stream-prefix": "awslogs-tis-template"
+          "awslogs-stream-prefix": "awslogs-tis-revalidation-concerns"
         }
       },
       "secrets": [
         {
           "name": "DB_HOST",
-          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-template-db-host"
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-revalidation-concerns-db-host"
         },
         {
           "name": "DB_PORT",
-          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-template-db-port"
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-revalidation-concerns-db-port"
         },
         {
           "name": "DB_USER",
-          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-template-db-user"
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-revalidation-concerns-db-user"
         },
         {
           "name": "DB_PASSWORD",
-          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-template-db-password"
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-revalidation-concerns-db-password"
```

Comment:
This is a flaw in the template, we should copy the needed secrets from tis-hee-revalidation i.e.
```
{
          "name": "MONGODB_HOST",
          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/reval-mongo-host"
        },
        {
          "name": "MONGODB_PORT",
          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/reval-mongo-port"
        },
        {
          "name": "MONGODB_USERNAME",
          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/reval-mongo-user"
        },
        {
          "name": "MONGODB_PASSWORD",
          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/reval-mongo-password"
        }
```

Reply:
Ok I will copy